go/utils/config/config.go

```diff
@@ -48,6 +49,12 @@ func ParseConfig(configFile string) *SubmitConfig {
 			glog.Errorf("load config %s error: %v\n", configFile, yamlErr)
 			return nil
 		}
+
+		var re = regexp.MustCompile(`(/|\\)*$`)
```

Comment:
Regex is slow, why not simply remove the trailing slash. By the way, can you add a verify of the endpoint string to ensure it is an URL?

Reply:
1. The user's input is diverse and only a few strings are checked.
2. Done!